jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -232,8 +232,6 @@ public ServicesCatalog createOrGetServicesCatalog(ClassLoader classLoader) {
 		return classLoader.createOrGetServicesCatalog();
 	}
 	
-	/* removed in build 160 */
-	@Deprecated
 	public ServicesCatalog getServicesCatalog(ClassLoader classLoader) {
```

Comment:
This this API is actually unused in the latest? As only getServicesCatalog(ModuleLayer) is used.